### PR TITLE
Update variable tokens

### PR DESCRIPTION
**Pull request by thiltunen**

### CODE REVIEW

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/49d3f0520572aa796e8676a1583fff7a6104c18e/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/3c92eba69f4794386552645986150799693afa02/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/8d2b2e948a0f6f271bbe41e482c33edb6d4818f0/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/2de14751b4446fa52a21c373e7e44676296ea648/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/9ba934e0cc9ba37d7fff35e24e0ef4f3ff7e9cf8/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/6c25ef5c60ece5c635fd577f3402b1dfd8ce8f25/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/98ab247c6bd70d741742cf384ea8ff0e6e872d64/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/2ba8b03d9ea61fdb2a78f199bccfb2137a44bd17/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/965b90fa6684e2a0d85f2c97f71a51ef5a3701ac/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/9a37571fb19fabd7ad3b18b270108d6f6436223c/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/5e963c082a46fc532bdf76071655dabdba774ace/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/fa1ff8f80c7496e2eb8f9d61455368aa186d2ad9/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/29f4ad392a0d6644e2c4ea5f238c1b47e420edf3/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/a3411fc20b3e375cb9dceb1d7f5d3d6779d8a031/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/0ce4a21bae13b31f6c73b266b80d51410984d65b/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/ac57f3589b43dbbf73331418d17ed26846220395/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/7ff42f56e889c4dbb80a658934d800d4a82e954d/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/e85abf922b589945867dc03eea263552f5c70736/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/f444bf2e67fd603b5114002af30423063b9251a8/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/4994ced6ce9dda24b9cd7c3c6daebfc12f21d29f/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

**Comment by giosg-machine-user** [comment]
You can download playwright screenshots [here](https://design-system-dev-screenshots.s3.amazonaws.com/drone/74c5e5a46996cfe2367876bdc39e53c84788b802/playwright_screenshots.tar.gz). If the "test-results" folder is empty, Playwright did not find any differences.

File: packages/foundations/README.MD [hunk 1]
```diff
@@ -13,3 +13,15 @@ Before you start the best thing you could do is to read the foundations package
 5. Update tests/screenshots if needed
 6. Commit your changes if you are fine with them
 7. Read the general rules on how to create changesets in [publishing settings](/README.md#Publishing)
+
+### ⁉️ Resolving-Referencing -ref- tokens.
+
+In the design-system we have two level's of nesting in `tokens.json` `ref` and `sys`. Reference level tokens are just building blocks to compose system level tokens. **They should not be used by users directly**.
```
Comment (thiltunen) [inline packages/foundations/README.MD:19]:
```suggestion
In the design-system we have two levels of nesting in `tokens.json` `ref` and `sys`. Reference level tokens are just building blocks to compose system level tokens. **They should not be used by users directly**.
```

Comment (gab1991) [reply packages/foundations/README.MD:19]: Ill fix it in the next pr 👍

**Review by gab1991 — APPROVED** [review]

**Pull request merged**